src/Analyzers/MSTest.Analyzers.CodeFixes/AddTestClassFixer.cs

```diff
@@ -49,7 +51,7 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
             CodeAction.Create(
                 title: CodeFixResources.PublicTypeShouldBeTestClassFix,
                 createChangedDocument: c => AddTestClassAttributeAsync(context.Document, declaration, c),
-                equivalenceKey: nameof(PublicTypeShouldBeTestClassFixer)),
+                equivalenceKey: nameof(AddTestClassFixer)),
```

Comment:
I don't know what behavior would we prefer, but doing it this way will cause a FixAll invocation for one diagnostic to fix things for both diagnostics. So not sure if that's desired from user point of view.